sdk/metric/provider.go

```diff
@@ -68,6 +69,10 @@ func NewMeterProvider(options ...Option) *MeterProvider {
 //
 // This method is safe to call concurrently.
 func (mp *MeterProvider) Meter(name string, options ...metric.MeterOption) metric.Meter {
+	if name == "" {
+		global.Warn("Invalid Meter name.", "meterName", name)
```

Comment:
I think from context the reader will know it is the meter name. Might be better to just leave it as "name"

```suggestion
		global.Warn("Invalid Meter name.", "name", name)
```

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/4153/commits/a401554e6678098565974abd7462bf9e50c93490